tests/test_modeling_common.py

```diff
@@ -2705,6 +2705,18 @@ def test_load_with_mismatched_shapes(self):
                     else:
                         new_model_without_prefix(input_ids)
 
+    def test_model_is_small(self):
+        # Just a consistency check to make sure we are not running tests on 80M parameter models.
+        config, _ = self.model_tester.prepare_config_and_inputs_for_common()
+        # print(config)
+
+        for model_class in self.all_model_classes:
```

Comment:
Do you think it's necessary to go through all `self.all_model_classes`, or just the base model class would be enough?

Reply:
All models, as one head might be big by mistake.